cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -116,6 +116,83 @@ func SendMessage(sent stores.Sent, senders map[string]sender.Message, ks keystor
 			return
 		}
 
+		receiver, ok := receivers[fmt.Sprintf("%s/%s", req.Protocol, req.Network)]
```

Comment:
This logic is needed to get new messages. This could be in "refresh messages function" a "POST" on "/messages" should work well.

Reply:
We are already using `POST` /messages for sending the messages. Shall we use `POST /messages/fetch` for refreshing the messages as an endpoint? So that we can separate them. I think in the task description, you mention that you would like to use `POST /messages` for getting the messages. That's why I have added the logic here 😅

Statements:
```
Create **POST /messages endpoint** and use the same with the same request parameters as /messages
New method will get message the same way as GET /messages however it will store them in the state database
```